src/main/java/io/wisetime/connector/jira/JiraDao.java

```diff
@@ -154,7 +154,13 @@ boolean canQueryDb() {
         .listResult(this::buildIssueFromResultSet);
   }
 
-  Optional<String> findUsername(final String email) {
+  Optional<String> findUserByUsername(final String username) {
+    return query().select("SELECT user_name FROM cwd_user WHERE lower_user_name = :username")
+        .namedParam("username", username.toLowerCase())
```

Comment:
Apparently, we can login to JIRA regardless of case we use for username, so let's mimic that behavior

Reply:
Can you add a comment to that effect?